package.json

```diff
@@ -105,11 +105,11 @@
     "tmp": "^0.2.1",
     "ts-node": "^10.7.0",
     "tslint": "^6.1.3",
-    "typescript": ">=3.3.1 <4.9.0"
+    "typescript": ">=3.3.1 <4.8.3 || 4.9.1-beta"
   },
   "resolutions": {
-    "typescript": "~4.8.3",
-    "@types/node": "^17.0.31",
+    "typescript": "~4.9.1-beta",
+    "@types/node": "^18.7.23",
```

Comment:
We need to bump `@types/node` because type errors on tests:

`"Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; }'.
`